### PR TITLE
Make 'com.google.android.gms:play-services-gcm' optional.

### PR DESCRIPTION
**Pull request by douglasjunior**
With this fix I can exclude the `play-services-gcm` dependency if needed.

Like this: (build.gralde)

```gradle
dependencies {
    compile(project(':react-native-device-info')) {
        exclude group: 'com.google.android.gms', module: 'play-services-gcm'
    }
    ....
} 
```

### CODE REVIEW

**Comment by GantMan** [comment]
interesting concept.  Maybe a friendly message tot he user?  Like setting the value to "N/A Add play services" would be best?

**Comment by douglasjunior** [comment]
Sounds good to me.

**Comment by douglasjunior** [comment]
Done.

File: RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -18,6 +18,8 @@ @implementation RNDeviceInfo
 
 @synthesize isEmulator;
 
+@synthesize isEmulator;
```
Comment (ex1machina) [inline RNDeviceInfo/RNDeviceInfo.m]: @GantMan this duplicate synthesize line is breaking the current release

Comment (GantMan) [reply RNDeviceInfo/RNDeviceInfo.m]: EEK!  PR?

Comment (GantMan) [reply RNDeviceInfo/RNDeviceInfo.m]: if not I can knock it out

Comment (ex1machina) [reply RNDeviceInfo/RNDeviceInfo.m]: @GantMan sorry about that, didn't have a chance to submit the PR, thanks for jumping on that so fast!

**Pull request merged**